corptools/src/index.js

```diff
@@ -0,0 +1,6 @@
+import { App } from "./App";
```

Comment:
What's going on in this file? I don't see anything exported, or `App` being used